x/nexus/types/msg_deactivate_chain.go

```diff
@@ -0,0 +1,56 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// NewDeactivateChainRequest creates a message of type DeactivateChainRequest
+func NewDeactivateChainRequest(sender sdk.AccAddress, chains ...string) *DeactivateChainRequest {
+	return &DeactivateChainRequest{
+		Sender: sender,
+		Chains: chains,
+	}
+}
+
+// Route implements sdk.Msg
+func (m DeactivateChainRequest) Route() string {
+	return RouterKey
+}
+
+// Type implements sdk.Msg
+func (m DeactivateChainRequest) Type() string {
+	return "DeactivateChain"
+}
+
+// ValidateBasic implements sdk.Msg
+func (m DeactivateChainRequest) ValidateBasic() error {
+	if err := sdk.VerifyAddressFormat(m.Sender); err != nil {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, sdkerrors.Wrap(err, "sender").Error())
+	}
+
+	if len(m.Chains) == 0 {
+		return fmt.Errorf("missing chains")
+	}
+
+	for _, chain := range m.Chains {
+		if chain == "" {
+			return fmt.Errorf("chain cannot be empty")
+		}
```

Comment:
```suggestion
		if err := utils.ValidateString(chain); err != nil {
			return sdkerrors.Wrap(err, fmt.Sprintf("invalid chain '%s'", chain))
		}
```

From here on, we should not forget to properly validate strings

Reply:
yep, haven't committed yet